packages/template-retail-react-app/app/components/swatch-group/swatch.jsx

```diff
@@ -57,9 +57,10 @@ const Swatch = ({
     return (
         <Button
             {...styles.swatch}
-            as={href ? RouteLink : 'button'}
+            as={href && !disabled ? RouteLink : 'button'}
```

Comment:
disabled prop only works for Button element, not the a element, so only use a tag if the component has href and not disabled

Reply:
I'm a bit confused here but I think I get it, so let me know if what I say below is correct or not. 

I think this is what we show on the PDP so we want to show un-orderable options as a disabled button.

But on the PLP product tile above, we don't show the button at all?

Reply:
Is this a breaking change?